connectivity/check/context.go

```diff
@@ -653,10 +653,21 @@ func (ct *ConnectivityTest) enableHubbleClient(ctx context.Context) error {
 			ct.Fail("In --flow-validation=strict mode, Hubble must be available to validate flows")
 			return fmt.Errorf("hubble is not available: %w", err)
 		}
-	} else {
-		ct.Infof("Hubble is OK, flows: %d/%d", status.NumFlows, status.MaxFlows)
+		return nil
 	}
 
+	for status.GetNumUnavailableNodes().GetValue() > 0 {
+		ct.Infof("Waiting for %d nodes to become available: %s",
+			status.GetNumUnavailableNodes().GetValue(), status.GetUnavailableNodes())
+		time.Sleep(5 * time.Second)
```

Comment:
Is there something special about 5s?

Reply:
no not really, i just picked a number and hardcoded it. i thought about adding a flag to make the retry interval configurable, but it felt like an overkill. what do you think? 💭